Tests/MongoSwiftTests/CrudTests.swift

```diff
@@ -276,7 +286,7 @@ private class DistinctTest: CrudTest {
 private class FindTest: CrudTest {
     override func execute(usingCollection coll: MongoCollection<Document>) throws {
         let filter: Document = try self.args.get("filter")
-        let options = FindOptions(batchSize: batchSize, collation: collation, limit: self.limit,
+        let options = FindOptions(batchSize: self.batchSize, collation: self.collation, limit: self.limit,
```

Comment:
happened to notice we weren't using `self`s here, which are unrequired but nice for clarity (and used everywhere else), so added them